libass/ass_fontselect.c

```diff
@@ -373,21 +373,52 @@ static void free_font_info(ASS_FontProviderMetaData *meta)
 /**
  * \brief Add a font to a font provider.
  * \param provider the font provider
- * \param meta basic metadata of the font
- * \param path path to the font file, or NULL
- * \param index face index inside the file (-1 to look up by PostScript name)
- * \param data private data for the font
- * \return success
+ * \param info The font to add to the database.
+ * \return True if the font has been successfully added to the provider. Otherwise, false.
+ * \note After calling this function, **do not** call `ass_font_provider_free_fontinfo`
+ *       on the `info` parameter, as its contents have been copied.
```

Comment:
nit, potential for confusion here: “its contents have been copied” is technically correct, but out of context when reading this I’d assume a (deep) copy was created meaning it would be safe to free the original.  
“as this takes ownership of its references” might be clearer